frame/collective/src/lib.rs

```diff
@@ -76,6 +76,30 @@ pub type MemberCount = u32;
 /// + This pallet assumes that dependents keep to the limit without enforcing it.
 pub const MAX_MEMBERS: MemberCount = 100;
 
+pub trait WeightInfo {
+	fn set_members(m: u32, n: u32, p: u32, ) -> Weight;
+	fn execute(m: u32, b: u32, ) -> Weight;
+	fn propose_execute(m: u32, b: u32, ) -> Weight;
+	fn propose_proposed(m: u32, p: u32, b: u32, ) -> Weight;
+	fn vote(m: u32, ) -> Weight;
+	fn close_early_disapproved(m: u32, p: u32, b: u32, ) -> Weight;
+	fn close_early_approved(m: u32, p: u32, b: u32, ) -> Weight;
+	fn close_disapproved(m: u32, p: u32, b: u32, ) -> Weight;
+	fn close_approved(m: u32, p: u32, b: u32, ) -> Weight;
+}
+
+impl WeightInfo for () {
+	fn set_members(_m: u32, _n: u32, _p: u32, ) -> Weight { 1_000_000_000 }
+	fn execute(_m: u32, _b: u32, ) -> Weight { 1_000_000_000 }
+	fn propose_execute(_m: u32, _b: u32, ) -> Weight { 1_000_000_000 }
+	fn propose_proposed(_m: u32, _p: u32, _b: u32, ) -> Weight { 1_000_000_000 }
+	fn vote(_m: u32, ) -> Weight { 1_000_000_000 }
+	fn close_early_disapproved(_m: u32, _p: u32, _b: u32, ) -> Weight { 1_000_000_000 }
+	fn close_early_approved(_m: u32, _p: u32, _b: u32, ) -> Weight { 1_000_000_000 }
+	fn close_disapproved(_m: u32, _p: u32, _b: u32, ) -> Weight { 1_000_000_000 }
+	fn close_approved(_m: u32, _p: u32, _b: u32, ) -> Weight { 1_000_000_000 }
```

Comment:
I do wonder whether we should rather default to 0 here because that seems like the current default. Either has its drawbacks I guess :shrug:

Reply:
yeah, its kind of just an opinion. im open to whatever, but my thought is that its better to stick a crazy number than to make it zero.